tg_bot/modules/blacklist.py

```diff
@@ -118,8 +118,9 @@ def del_blacklist(bot: Bot, update: Update):
 
     chat_filters = sql.get_chat_blacklist(chat.id)
     for trigger in chat_filters:
-        pattern = r"( |^|[^\w])" + re.escape(trigger) + r"( |$|[^\w])"
-        if re.search(pattern, to_match, flags=re.IGNORECASE):
+      pattern = r"( |^|[^\w])" + trigger + r"( |$|[^\w])"
+      try:
+        if re.search(pattern, to_match, flags=re.IGNORECASE).group():
```

Comment:
Same like in filters, re.search(..) will give you none and will pas the check. So there's no use for it or try: except:

Reply:
It causes some annoying issue, donno why it even matched on some cases, that's why I used group. I knew it returns None, but on some cases some empty match strings, which upon group() sends a None.